node/rpc/src/lib.rs

```diff
@@ -130,14 +132,15 @@ where
 	)));
 	io.extend_with(Web3ApiServer::to_delegate(Web3Api::new(client.clone())));
 	io.extend_with(EthPubSubApiServer::to_delegate(EthPubSubApi::new(
-		pool,
-		client,
+		pool.clone(),
```

Comment:
```suggestion
		Arc::clone(&client),
```
See clippy lint : https://rust-lang.github.io/rust-clippy/master/index.html#clone_on_ref_ptr

Reply:
Probably part of another PR that takes care of convention in all rpc extensions at service level.